apis/v1beta1/shared_types.go

```diff
@@ -246,27 +246,25 @@ type PortNumber int32
 //
 // <gateway:experimental:description>
 //
-// When the BackendRef points to a Kubernetes Service, implementations SHOULD honor the
-// appProtocol field if it is set for the target Service Port.
+// When the BackendRef points to a Kubernetes Service, implementations SHOULD
+// honor the appProtocol field if it is set for the target Service Port.
 //
-// Implementations supporting appProtocol SHOULD recognize the Kubernetes Standard Application Protocols
-// defined in [KEP-3726].
+// Implementations supporting appProtocol SHOULD recognize the Kubernetes
+// Standard Application Protocols defined in KEP-3726.
 //
-// If a Service appProtocol isn't specified an implementation MAY infer the backend
-// protocol through its own means. Implementations MAY infer the protocol from the
-// Route type referring to the backend Service.
+// If a Service appProtocol isn't specified, an implementation MAY infer the
+// backend protocol through its own means. Implementations MAY infer the
+// protocol from the Route type referring to the backend Service.
 //
-// If a Route is not able to send traffic to the backend using the specified protocol then
-// the backend is considered invalid. Implementations MUST set ResolvedRefs condition to
-// False with the "UnsupportedProtocol".
+// If a Route is not able to send traffic to the backend using the specified
+// protocol then the backend is considered invalid. Implementations MUST set the
+// "ResolvedRefs" condition to "False" with the "UnsupportedProtocol" reason.
 //
 // </gateway:experimental:description>
 //
 // Note that when the BackendTLSPolicy object is enabled by the implementation,
 // there are some extra rules about validity to consider here. See the fields
 // where this struct is used for more information about the exact behavior.
-//
-// [KEP-3726]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/3726-standard-application-protocols
```

Comment:
For the record, I initially tried to keep this link but just move it up above the `</gateway:experimental:description>` tag, but then `gofumpt` failed. I think not having a link here is better than this link mysteriously appearing in standard without context, so went with that, but open to any follow ups to add this back in only for experimental.

Reply:
Yeah I had the same problem